rpc/amqp-rpc.go

```diff
@@ -599,7 +580,7 @@ func (rpc *AmqpRPCCLient) SetTimeout(ttl time.Duration) {
 // Dispatch sends a body to the destination, and returns a response channel
 // that can be used to monitor for responses, or discarded for one-shot
 // actions.
-func (rpc *AmqpRPCCLient) Dispatch(method string, body []byte) chan []byte {
+func (rpc *AmqpRPCCLient) dispatch(method string, body []byte) chan []byte {
```

Comment:
Oh, missed this. There are some docs in this file that refer to Dispatch. They should probably refer to DispatchSync now.